configure.ac

```diff
@@ -884,6 +884,15 @@ if test x$use_reduce_exports = xyes; then
   [AC_MSG_ERROR([Cannot set default symbol visibility. Use --disable-reduce-exports.])])
 fi
 
+AC_MSG_CHECKING([for std::system])
+AC_TRY_LINK( [ #include <cstdlib> ],
```

Comment:
I see these warnings when running `./autogen.sh`:
```
configure.ac:894: warning: The macro `AC_TRY_LINK' is obsolete.
configure.ac:894: You should run autoupdate.
../../lib/autoconf/general.m4:2688: AC_TRY_LINK is expanded from...
configure.ac:894: the top level
glibtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, 'build-aux'.
...
```

Reply:
I switched to the modern `AC_LINK_IFELSE( [ AC_LANG_PROGRAM(` syntax. For some reason I don't think I got these warnings on macOS.